code/modules/clothing/shoes/colour.dm

```diff
@@ -1,104 +1,113 @@
-/obj/item/clothing/shoes/black
-	name = "black shoes"
-	icon_state = "black"
-	desc = "A pair of black shoes."
+/obj/item/clothing/shoes/color
+	name = "white shoes"
+	icon_state = "white"
+	desc = "A pair of shoes."
 
 	cold_protection = FEET
 	min_cold_protection_temperature = SHOE_MIN_COLD_PROTECTION_TEMPERATURE
 	heat_protection = FEET
 	max_heat_protection_temperature = SHOE_MAX_HEAT_PROTECTION_TEMPERATURE
 
-/obj/item/clothing/shoes/brown
+	var/markings_icon = "shoes_coloring"
+	var/markings_color
+
+/obj/item/clothing/shoes/color/Initialize()
+	. = ..()
+	update_icon()
+
+/obj/item/clothing/shoes/color/on_update_icon()
+	if(markings_color)
+		overlays.Cut()
```

Comment:
Might be worth using `cut_overlays()` and `add_overlay()` since you're redefining this entirely here.